[2023-01-08T15:07:08.691+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: atn_update_datalake.add_init_step_to_second_emr manual__2023-01-08T15:06:09.206673+00:00 [queued]>
[2023-01-08T15:07:08.704+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: atn_update_datalake.add_init_step_to_second_emr manual__2023-01-08T15:06:09.206673+00:00 [queued]>
[2023-01-08T15:07:08.704+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-01-08T15:07:08.704+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 3
[2023-01-08T15:07:08.704+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-01-08T15:07:08.718+0000] {taskinstance.py:1389} INFO - Executing <Task(EmrAddStepsOperator): add_init_step_to_second_emr> on 2023-01-08 15:06:09.206673+00:00
[2023-01-08T15:07:08.723+0000] {standard_task_runner.py:52} INFO - Started process 329 to run task
[2023-01-08T15:07:08.726+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'atn_update_datalake', 'add_init_step_to_second_emr', 'manual__2023-01-08T15:06:09.206673+00:00', '--job-id', '4053', '--raw', '--subdir', 'DAGS_FOLDER/market_data_analytics/my_dags/datalake_update.py', '--cfg-path', '/tmp/tmpm__sqsfd', '--error-file', '/tmp/tmp5_1cyp4l']
[2023-01-08T15:07:08.726+0000] {standard_task_runner.py:80} INFO - Job 4053: Subtask add_init_step_to_second_emr
[2023-01-08T15:07:08.792+0000] {task_command.py:371} INFO - Running <TaskInstance: atn_update_datalake.add_init_step_to_second_emr manual__2023-01-08T15:06:09.206673+00:00 [running]> on host 370ed6868bc3
[2023-01-08T15:07:08.898+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=atn
AIRFLOW_CTX_DAG_ID=atn_update_datalake
AIRFLOW_CTX_TASK_ID=add_init_step_to_second_emr
AIRFLOW_CTX_EXECUTION_DATE=2023-01-08T15:06:09.206673+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-08T15:06:09.206673+00:00
[2023-01-08T15:07:08.908+0000] {base.py:68} INFO - Using connection ID 'aws_credentials' for task execution.
[2023-01-08T15:07:08.909+0000] {connection_wrapper.py:210} INFO - AWS Connection (conn_id='aws_credentials', conn_type='aws') credentials retrieved from login and password.
[2023-01-08T15:07:08.970+0000] {emr.py:104} INFO - Adding steps to j-36MHGJC6JXK5R
[2023-01-08T15:07:09.957+0000] {emr.py:117} INFO - Steps ['s-32PNMSF5A4XS8', 's-2UBFDIB2JJ531', 's-2DV1WYLLTQES7', 's-LA7F46KC2ACK', 's-P2HU3I3YG1C8', 's-ND8PWKOOEWL6', 's-I8UPQFCTSAX0'] added to JobFlow
[2023-01-08T15:07:10.008+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=atn_update_datalake, task_id=add_init_step_to_second_emr, execution_date=20230108T150609, start_date=20230108T150708, end_date=20230108T150710
[2023-01-08T15:07:10.069+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-08T15:07:10.146+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
